Tools/Scripts/webkitdirs.pm

```diff
@@ -599,6 +581,56 @@ sub determineArchitecture
     $architecture = 'arm64' if $architecture =~ /aarch64/i;
 }
 
+sub determineDestination
```

Comment:
`detemineXcodeDestination`, to make it more obvious that this doesn't apply to other platforms?